nexus/mgs-updates/src/host_phase1_updater.rs

```diff
@@ -3,20 +3,153 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 //! Module containing types for updating host OS phase1 images via MGS.
+//!
+//! Reconfigurator-driven OS updates are complicated because it has to
```

Comment:
This seems like a good spot to explain (maybe briefly) about phase1/phase2 and slot A/slot B.

Reply:
Added in 06fde0f6f